### PR TITLE
Fix README and top level gradle file to support benchmarks.

### PR DESCRIPTION
**Pull request by cmelchior**
@realm/java

### CODE REVIEW

**Comment by beeender** [comment]
:+1:  I guess we need review from @emanuelez

File: README.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ That command will generate:
  * `./gradlew monkeyExamples` will run the monkey tests on all the examples
  * `./gradlew installRealmJava` will install the Realm library and plugin to mavenLocal()
  * `./gradlew clean -PdontCleanJniFiles` will remove all generated files except for JNI related files. This saves recompilation time a lot.
- * `./gradlew connectedCheck -PbuildTargetABIs=$(adb shell getprop ro.product.cpu.abi)` will build JNI files only for the ABI which corresponds to the connected device.
+ * `./gradlew connectedUnitTests -PbuildTargetABIs=$(adb shell getprop ro.product.cpu.abi)` will build JNI files only for the ABI which corresponds to the connected device.
```
Comment (kneth) [inline README.md]: I guess that it is assumed that you only have one device connected.

**Comment by kneth** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (cmelchior) [inline README.md]: Yes, the behaviour hasn't changed in this regard.

**Comment by dalinaum** [comment]
:+1:

**Pull request merged**